contracts/extensions/votingReputation/VotingReputationDataTypes.sol

```diff
@@ -40,6 +41,13 @@ interface VotingReputationDataTypes {
     bool finalized;
     address altTarget;
     bytes action;
+    bytes4 sig;
```

Comment:
I added it to the end because otherwise I worried it would have messed up the storage layout for existing motions. When we release a proper V2 we can re-organize the storage layout. Unless you don't think there's an issue?

Reply:
There shouldn't be, based on my understanding. How do you think it's going to change the storage layout?

Reply:
If we insert it into the existing storage layout, wouldn't it push existing data down a slot? The only place I could see it going is after `altTarget`, where it would get packed after existing data. Do you agree?

Reply:
I've update the layout as you requested and tested to make sure upgrades don't mess up the layout.